.travis.yml

```diff
@@ -4,10 +4,12 @@ node_js:
 cache:
   directories:
     - node_modules
+before_install:
+  - npm install -g npm@latest
```

Comment:
Why is this needed? Travis will have node 6 with npm 3 since node v6 is in this config. Adding this extra steps makes travis take longer and can potentially break things when npm has a new major version release.